source/production/F0.Minesweeper.Components/Pages/Game/Game.razor.cs

```diff
@@ -31,6 +33,8 @@ private void OnDifficultyChanged(DifficultyLevel selectedDifficulty)
 			StateHasChanged();
 		}
 
+		private void RestartGame() => StateHasChanged();
```

Comment:
Should we prefix this method with `On` as well.

Reply:
Added the `On` prefix since it's kind of a convention to use that.